pkg/controller/backup_sync_controller.go

```diff
@@ -232,9 +238,38 @@ func (c *backupSyncController) run() {
 			default:
 				log.Debug("Synced backup into cluster")
 			}
+
+			// process the pod volume backups from object store, if any
+			podVolumeBackups, err := backupStore.GetBackupPodVolumes(backupName)
+			if !kuberrs.IsNotFound(err) {
+				log.WithError(errors.WithStack(err)).Error("Error getting pod volumes for this backup, proceeding with sync into cluster")
+			}
+
+			if kuberrs.IsNotFound(err) {
+				log.WithError(errors.WithStack(err)).Error("Error getting pod volumes for this backup")
+				continue
+			}
+
+			for _, podvolumeBackup := range podVolumeBackups {
+				log = log.WithField("podVolumeBackup", podvolumeBackup.Name)
+				log.Debug("Checking this pod volume to see if it needs to be synced into the cluster")
+
+				_, err = c.podVolumeBackupClient.PodVolumeBackups(backup.Namespace).Create(podvolumeBackup)
+				switch {
+				case err != nil && kuberrs.IsAlreadyExists(err):
+					log.Debug("Pod volume already exists in cluster")
+					continue
+				case err != nil && !kuberrs.IsAlreadyExists(err):
+					log.WithError(errors.WithStack(err)).Error("Error syncing pod volume into cluster")
+					continue
+				default:
+					log.Debug("Synced pod volume into cluster")
+				}
+			}
 		}
 
 		c.deleteOrphanedBackups(location.Name, backupStoreBackups, log)
+		c.deleteOrphanedPodVolumeBackups(location.Name, backupStoreBackups, log)
```

Comment:
I'm not sure that this function is needed -- `PodVolumeBackups` are linked by owner ref to the `Backup` that triggered them, and when the `Backup` is deleted, the owned `PodVolumeBackups` are automatically deleted as well. Need to think about it a little more.

Reply:
Ok, let me know. I'm not familiar with how the owner ref, probably a good opportunity for me to look into it too.

Reply:
The owner reference is set on the metadata, had to look this up myself. https://github.com/heptio/velero/blob/master/pkg/restic/backupper.go#L231

Reply:
See also https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#ownerreference-v1-meta

Reply:
@skriss  So.. `The owner reference is set on the metadata`, and the metadata has the backup info, deleting the backup would delete any `podvolumebackup` associated with it?

Reply:
Yep - that's correct. So (a) we shouldn't have a scenario where a PVB exists that doesn't have a Backup as an owner, and (b) deleting a backup will automatically delete associated PVBs. So as far as I can tell, we shouldn't have a need for this function

Reply:
Thank you 👍